### DISCUSSION TITLE
Use library with external I2S initialization

### DISCUSSION THREAD

**Discussion by rbuehlma** [pschatzmann/ESP32-A2DP#338] [Q&A] [answered]
I would like to use this library with the I2S driver already initialized. For this, the call to init_i2s and i2s_driver_uninstall should be skipped.

Is it possible to add an option to achieve this?

**Comment by pschatzmann** [comment, marked as answer]
I dont consider this an issue because the library is doing what it is supposed to do.
I also assume that you did not read the Design Overview which describes how you can achieve this with just a few lines of code!

From a design point of view I offer the choice, that the library manages I2S or gives you the choice to do it yourself: which gives you all the flexibility you need. After all combined with my audio-tools library this can be done with only a few lines of code!

**Comment by rbuehlma** [comment]
You are right, this is not an issue and I should be able to implement it by overriding init_i2s and i2s_driver_uninstall.
